modules/ROOT/pages/runtime-fabric-limitations.adoc

```diff
@@ -0,0 +1,40 @@
+= Limitations
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The following information is important when installing, configuring, 
+and managing Runtime Fabric.
+
+== Controller Nodes
```

Comment:
Consider using a DL for these, since they're mostly 1 sentence long?

Reply:
Yes - this looks better - thanks!